src/AutoRest.CSharp.V3/Output/Models/Requests/MultipartRequestBody.cs

```diff
@@ -0,0 +1,22 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.Collections.Generic;
+using System.Reflection.Metadata;
+
+namespace AutoRest.CSharp.V3.Output.Models.Requests
+{
+    internal class MultipartRequestBody : RequestBody
+    {
+        public ReferenceOrConstant Value { get; }
```

Comment:
I've expected to see an array of parts here. Why do we only have 2 properties?

Reply:
Changed to array.